content/learn/case-studies/cnbc.md

```diff
@@ -0,0 +1,50 @@
+---
+$order: 14
+$title: CNBC
+$date: May 22, 2017
+$category: Publishers
+$parent: /content/learn/case-studies.html
+class: case-study post-detail
+
+components:
+  - social-share
+
+headline: CNBC sees mobile speed and engagement gains with AMP
```

Comment:
Typically the headline matches the one in the PDF.  Is there a reason for the difference?